database/migrations/2020_09_02_033245_add_bundled_flags_to_beatmapsets.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddBundledFlagsToBeatmapsets extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('osu_beatmapsets', function (Blueprint $table) {
+            $table->boolean('bundled')->default(false);
+            $table->boolean('can_bundle')->default(false);
+            $table->index('bundled');
+            $table->index('can_bundle');
```

Comment:
these indices seem to be no use now?

Reply:
used in #6634, sorry this is all over the place ;_ ; ended up in this PR cuz i thought i needed it